scripts/buildCss.js

```diff
@@ -0,0 +1,41 @@
+// core packages
+const fs = require('fs'),
+      path = require('path')
+
+// npm packages
+const simpleIcons = require('simple-icons')
+
+
+const basePath = path.join(__dirname, '..')
+
+const attributedIcons = Object.values(simpleIcons).map(icon => {
+	return {
+		color: icon.hex,
+		cssClass: icon.title
+			.toLowerCase()
+			.replace(/[ !’]/g, '')
+			.replace(/-/g, '_') // hyphens not supported in ligatures
+			.replace(/\./g, 'dot') // dot not supported in css class names
+			.replace(/\+/g, 'plus') // plus not supported in css class names
+			.replace(/&/g, 'and') // ampersand not supported in css class names
```

Comment:
You use the exact same transformation in `buildTestPage.js`, might be a good idea to add this to the utilities file